source/scpd/types/Stellar_types.d

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+
+    Porting of Stellar's `Stellar_types.h`, itself derived from
+    `Stellar_types.x`
+
+    Copyright:
+        Copyright (c) 2019 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module scpd.types.Stellar_types;
```

Comment:
In modules like this, we should generate `static assert` which check the size of the types...